.github/workflows/integrations.yml

```diff
@@ -10,9 +10,13 @@ on:
     paths-ignore:
       - 'docs/**'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  test:
-    name: ${{ matrix.config.os }}
+  python-e2e:
```

Comment:
My thought was more what component its testing, rather than the installation method.  I thought to call out that the e2e tests are limited to the Python-only API, and the new test is essentially just testing CMake packaging.

Is there another name that would be less confusing to capture that?

Reply:
Ah other comment https://github.com/OpenAssetIO/OpenAssetIO/pull/723#discussion_r1016252685 also covers this.